docs/source/intro/01-install.md

```diff
@@ -1,26 +1,24 @@
 # Installing Deluge
 
-These are the instructions for installing Deluge. Consider them a work-in-progress and
-feel free to make suggestions for improvement.
+Instructions for installing Deluge server and the GTK client.
```

Comment:
There is no mention of server or client in this document so simply say `Deluge`

Reply:
I found that confusing. I installed `deluge`, then pressed super, typed "deluge", but nothing appeared. It was because only the server was installed. I think it's important to mention both.

Reply:
Whoops my mistake. That only happened when installing the snap from Ubuntu's software app. The first entry is a snap, the 2nd is from apt.
![image](https://user-images.githubusercontent.com/66286679/84597255-da28ed80-ae30-11ea-8d93-b700f3f8f4f9.png)

Reply:
Speaking of, what is this?
https://github.com/Lukewh/deluge-snap